src/autoscaler/eventgenerator/aggregator/metric_poller.go

```diff
@@ -63,7 +63,7 @@ func (m *MetricPoller) retrieveMetric(appMonitor *models.AppMonitor) error {
 	startTime := endTime.Add(0 - statWindow)
 
 	metrics, err := m.metricClient.GetMetrics(appId, metricType, startTime, endTime)
-	m.logger.Debug("received metrics from metricClient:", lager.Data{"retrivedMetrics": metrics})
```

Comment:
Not sure why several message contain `:`. 

It doesn't make sense to have `:` in the messages, example:
```
{
  "data": {
   ...
  },
  "log_level": 0,
  "log_time": "2024-03-22T10:03:52Z",
  "message": "eventgenerator.MetricPoller.received metrics from metricClient:", ⬅️ 😲
  "source": "eventgenerator"
}
```